spec/controllers/application_controller_spec.rb

```diff
@@ -122,4 +122,12 @@ def index
       end
     end
   end
+
+  describe '#analytics' do
+    it 'uses the Analytics class by default' do
+      expect(Analytics).to receive(:new).with(controller.current_user, request)
```

Comment:
I don't think so. In this case, we want to know specifically that the correct arguments were passed. We always want the current user and request to be passed as arguments.